docs/Change_Log.md

```diff
@@ -1,5 +1,24 @@
 # Change Log
 
+## API and Gateway V8
+
+#### September 24, 2020
+
+We've introduced API and Gateway v8! Changes are noted throughout the documentation, but the major changes are:
+- API v8 is now default
```

Comment:
v8 is not yet default. we'll need to have a deprecation period before it hits default